invisible_cities/database/load_db_test.py

```diff
@@ -178,3 +179,64 @@ def test_pmt_noise_frequencies(db):
     assert frequencies.shape[0] == db.nfreqs
     assert frequencies.shape[1] == db.feboxes + 1
     assert np.all(frequencies[:, 0] == freq_expected)
+
+
+def test_RadioactivityData_max_version():
+    '''Check that RadioactivityData returns by default the data with maximum Version number'''
+
+    db_file = "next100"
+
+    # test unique (G4Volume, Isotope)
+    activity, efficiency = DB.RadioactivityData(db_file)
+    assert np.all(activity  .groupby(["G4Volume", "Isotope"]).count() == 1)
+    assert np.all(efficiency.groupby(["G4Volume", "Isotope"]).count() == 1)
+
+    conn = sqlite3.connect(DB.get_db(db_file))
+
+    # test Activity table
+    query = '''SELECT * FROM Activity
+               WHERE G4Volume = '{0}' AND Isotope = '{1}'
+            '''
+    for (g4volume, isotope), req in activity.groupby(["G4Volume", "Isotope"]):
+        all_versions = pd.read_sql_query(query.format(g4volume, isotope), conn)
+        max_version = all_versions[all_versions.Version == all_versions.Version.max()]
+        np.testing.assert_allclose(req.TotalActivity, max_version.TotalActivity)
+
+    # test Efficiency table
+    query = '''SELECT * FROM Efficiency
+               WHERE G4Volume = '{0}' AND Isotope = '{1}'
+            '''
+    for (g4volume, isotope), req in efficiency.groupby(["G4Volume", "Isotope"]):
+        all_versions = pd.read_sql_query(query.format(g4volume, isotope), conn)
+        max_version = all_versions[all_versions.Version == all_versions.Version.max()]
+        np.testing.assert_allclose(req.MCEfficiency, max_version.MCEfficiency)
```

Comment:
loop to avoid code repetition? Same for the other test. Review alignment and I think we are done.

Reply:
Not sure how is the best way to do it or if it is clearer, since the column name is different for Activity table (TotalActivity) and Efficiency (MCEfficiency). Also, (g4volume, isotope) might be missing, we are not testing that both tables contain all the combinations, which for example is not the case in the current dummy tables.

Reply:
I was referring to something like
```suggestion
    for table, table_name, column in zip((      activity ,    efficiency ),
                                         (     "Activity",   "Efficiency"),
                                         ("TotalActivity", "MCEfficiency")):

        query = "SELECT * FROM {0} WHERE G4Volume = '{1}' AND Isotope = '{2}'"
        for (g4volume, isotope), req in table.groupby(["G4Volume", "Isotope"]):
            all_versions = pd.read_sql_query(query.format(table_name, g4volume, isotope), conn)
            max_version = all_versions[all_versions.Version == all_versions.Version.max()]
            np.testing.assert_allclose(req[column], max_version[column])
```
I think the lack of repetition compensates for the slightly less explicit code.

I don't know what do you mean by
> (g4volume, isotope) might be missing

Reply:
Ok, that looks good to me. I was discarting to do the column call like `req[column]` since you requested above the change of sintax `df["column_name"]` to `df.column_name`.

Reply:
It's a bit of a trade-off. I definitely prefer using attributes, but that's not so handy when the name is stored in a variable. So as a general rule, use attribute access for hard-coded values and item access for variables.